Gems/Atom/Feature/Common/Code/Include/Atom/Feature/PostProcess/ChromaticAberration/ChromaticAberrationParams.inl

```diff
@@ -0,0 +1,21 @@
+/*
```

Comment:
Maybe not necessarily necessary for this PR, but I think a more common pattern would be defining this as a header which declares macros, and then using those macros in code, as opposed to the pattern in this PR which is to expand these `.inl` files inline the file.

The benefit of the explicit macro definition and call is that it's maybe slightly more clear what's happening, and there's no risk that a person modifying this file inadvertently changes the interface or struct layout or something of a class that expands it.

Reply:
> Maybe not necessarily necessary for this PR, but I think a more common pattern would be defining this as a header which declares macros, and then using those macros in code, as opposed to the pattern in this PR which is to expand these .inl files inline the file.

> The benefit of the explicit macro definition and call is that it's maybe slightly more clear what's happening, and there's no risk that a person modifying this file inadvertently changes the interface or struct layout or something of a class that expands it.

I think you can blame @antonmic for this setup.

Reply:
I was aiming to stick as close as possible to other examples so any original author would recognise the pattern. This comment applies to some of the other issues as well.